### ISSUE TITLE
Font Library modal: Improve clarity between active and inactive fonts

### ISSUE THREAD

**Issue by mikachan** [WordPress/gutenberg#58276]
## What problem does this address?

In the Library tab of the Font Library modal, it's not very clear when a font is active or inactive:
<img width="600" alt="Screenshot 2024-01-25 at 19 33 43" src="https://github.com/WordPress/gutenberg/assets/1645628/404eef28-80c3-4e7c-b227-b1d561e16cce">

## What is your proposed solution?

Should we consider adding a badge or icon to indicate state?

cc. @jasmussen

**Comment by jasmussen** [comment]
Instead of `0/1 variants active`, we can reword it to `0 of 1 active`, and have it be $gray-900 in full dark color.

The motivation for this isn't that clear, though; what's the main problem this is meant to solve? Your active fonts will also show up here, which is the first thing you'll see:

![Screenshot 2024-01-26 at 09 25 05](https://github.com/WordPress/gutenberg/assets/1204802/5cac36b4-db14-450e-ad1e-47fae420ce10)

**Comment by colorful-tones** [comment]
Probably adding in some scope creep, but I feel like these related issues could likely get a 'Needs Design' flow and consideration overall. @jasmussen 

- [Font Library: Global Styles > Typography sidebar – group related font sources #58403](https://github.com/WordPress/gutenberg/issues/58403)
- [Font Library: Consider clarifying which fonts are uploaded vs theme provided in drop down #55220](https://github.com/WordPress/gutenberg/issues/55220)

**Comment by karmatosed** [comment]
I think this should just have a feedback label as looks like we have a pretty solid suggestion.
